pages/dkp/kommander/1.4/projects/project-deployments/index.md

```diff
@@ -4,26 +4,26 @@ beta: true
 navigationTitle: Project Deployments
 title: Project Deployments
 menuWeight: 1
-excerpt: Project Deployments can be used to manage GitOps based Continuous Deployments which can automatically be created on all the Kubernetes clusters associated with the Project, in the corresponding namespace.
+excerpt: Use Project Deployments to manage GitOps based Continuous Deployments.
 ---
 
-Kommander Projects can be configured with GitOps based Continuous Deployments for federation of your Applications to associated clusters of the project. This is backed by Dispatch which enables software and applications to be continuously deployed (CD) using GitOps processes. GitOps enables the application to be deployed as per a manifest that is stored in a Git repository. This ensures that the application deployment can be automated, audited and declaratively deployed to the infrastructure.
+Kommander Projects are configured with GitOps based Continuous Deployments for federation of your Applications to associated clusters of the project. This is backed by Dispatch which enables software and applications to be continuously deployed (CD) using GitOps processes. GitOps enables the application to be deployed as per a manifest that is stored in a Git repository. This ensures that the application deployment can be automated, audited and declaratively deployed to the infrastructure.
```

Comment:
Do you have to use CD?  The "can be"->"are" change here suggests that CD is the only way "Kommander Projects are configured".

Reply:
Good point.